plans/chat-mobile-plan/05-uc-comp.md

```
@@ -0,0 +1,89 @@
+functional_group: COMP
+---
+
+# Use Cases: Composition + Send (COMP)
+
+| ID | Title | Description |
+|----|-------|-------------|
+| UC-COMP-01 | Compose a message in Tiptap editor | User can type multiline text in the chat input with placeholder, autogrow, and slash/mention atomic tokens via `@10play/tentap-editor` for desktop parity. |
+| UC-COMP-02 | Submit message with optimistic append | User can tap Send to submit a message and see it appear in the message list before the host acknowledges. |
+| UC-COMP-03 | Stop a running turn | User can interrupt an in-progress assistant turn from the composer area. |
+| UC-COMP-04 | Pick the model for a turn | User can choose between available models (Opus, Sonnet, Haiku, GPT variants) from a popover. |
+| UC-COMP-05 | Set thinking level and permission mode | User can adjust thinking level (`off | low | medium | high | xhigh`) and permission mode from popovers in the composer. |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 7; Too many cells, extra data will be missing

(MD056, table-column-count)